crates/typst-cli/Cargo.toml

```diff
@@ -37,7 +37,7 @@ same-file = "1"
 # https://github.com/mitsuhiko/self-replace/pull/16
 self-replace = { git = "https://github.com/Gui-Yom/self-replace" }
 semver = "1"
-serde = { version = "1", features = ["derive"] }
+serde = ">= 1.0.126, <= 1.0.171"
```

Comment:
1.0.184 removes the pre-compiled binary. I would set that as the new minimum to allow continued updates but make sure you skip the problematic versions.

Reply:
How is that related to this PR?

Reply:
Anyway, I pushed the serde change to main.

Reply:
Presumably it isn't, the snafu with serde just being something that happened to come up during the lifespan of the PR. I would suggest it get addressed separately (by adding a minimum version past the version with binary blobs) and rebase this PR to stay focused ... unless this PR was what introduced the dependency in the first place in which case it should be fixed here.

Reply:
It's already fixed on main.